packages/enzyme-test-suite/test/ShallowWrapper-spec.jsx

```diff
@@ -1273,6 +1275,45 @@ describe('shallow', () => {
         expect(wrapper.find(Component.displayName)).to.have.lengthOf(2);
       });
     });
+
+    describeIf(is('>= 16.6'), 'Suspense & lazy', () => {
+      it('finds Suspense and its children when no lazy component', () => {
+        class Component extends React.Component {
+          render() {
+            return (
+              <div>test</div>
+            );
+          }
+        }
+        const SuspenseComponent = () => (
+          <Suspense fallback={false}>
+            <Component />
+          </Suspense>
+        );
+
+        const wrapper = shallow(<SuspenseComponent />);
+
+        expect(wrapper.find('Suspense')).to.have.lengthOf(1);
```

Comment:
Since a shallow wrapper is "what the wrapped thing *renders*", i would expect this to be:
```suggestion
        expect(wrapper.is(Suspense)).to.equal(true);
```